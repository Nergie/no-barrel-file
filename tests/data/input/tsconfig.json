{
  "compilerOptions": {
    "baseUrl": "./",
    "jsx": "preserve",
    "paths": {
      "@barrel-basic/*": ["./barrel-basic/*"],
      "@barrel-circular/*": ["./barrel-circular/*"],
      "@barrel-nested/*": ["./barrel-nested/*"],
      "@ignored/*": ["./ignored/*"],
    }
  }
}
