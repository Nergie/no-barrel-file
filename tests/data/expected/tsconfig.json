{
  "compilerOptions": {
    "baseUrl": "./",
    "jsx": "preserve",
    "lib": [
      "esnext",
      "dom"
    ],
    "paths": {
      "@barrel-basic/*": ["./barrel-basic/*"],
      "@barrel-circular/*": ["./barrel-circular/*"],
      "@barrel-nested/*": ["./barrel-nested/*"],
      "@ignored/*": ["./ignored/*"],
      "@barrel-very-nested/*": ["./barrel-very-nested/*"],
      "@feature-a/*": ["./barrel-path-alias-complex/libs/feature-a/src/*"],
      "@feature-b/*": ["./barrel-path-alias-complex/libs/feature-b/src/*"],
      "@shared/*": ["./barrel-path-alias-complex/libs/shared/src/*"],
      "@utils/*": ["./barrel-path-alias-complex/utils/*"],
      "~components/*": ["./barrel-path-alias-complex/components/*"],
      "~types/*": ["./barrel-path-alias-complex/types/*"]
    }
  }
}
